kplclient.go

```diff
@@ -62,6 +77,57 @@ func processChannel() {
 	}
 }
 
+func (c *KPLClient) processErrMessage() {
+	l, err := net.Listen("tcp", c.ErrPort)
```

Comment:
IMO `kplserver` is the process that should publish the err socket, and `kplclient` should _connect_ to it (a la `Dial()`). One of the reasons for this is that it would enable multiple clients to subscribe to those error messages.

P.S. What we should _really_ do, is something more sophisticated (perhaps with ZMQ) where on initial connection the server provides an ephemeral return socket so that each client gets its own dedicated communication channel... but that's a task for another day!

Reply:
>One of the reasons for this is that it would enable multiple clients to subscribe to those error messages.

I was actually trying to avoid that possibility. My thinking was that we should not allow multiple people to receive the same error as only one client would have given the message that failed. Any advantage you see in that Lane?

Reply:
I definitely see your point. I guess the first question I should ask is, can multiple clients currently connect/publish to kplserver?

Reply:
That is a good point. kplserver allows only one client to connect.

Reply:
Hmm that's annoying. But I suppose as long as we're not regressing on any of that functionality then what you have here makes sense.